pkgs/by-name/se/seadrive-gui/package.nix

```diff
@@ -0,0 +1,53 @@
+{ cmake
+, fetchFromGitHub
+, pkg-config
+, stdenv
+, lib
+, extra-cmake-modules
+# Package dependencies
+, libsepol
+, seadrive-fuse
+, qt5
+}:
+
+stdenv.mkDerivation rec {
+  pname = "seadrive-gui";
+  version = "2.0.16";
+
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = "seadrive-gui";
+    rev = "v${version}";
+    sha256 = "0gi2l2knyl0903fslwdhkqlm3p88dznk5nns9lfimbk8as8qd0mf";
+  };
+
+  patches = [ ./fix_webkit_webengine.patch ];
+
+  nativeBuildInputs = [
+    qt5.wrapQtAppsHook
+    cmake
+    pkg-config
+    qt5.qttools
+    extra-cmake-modules
+  ];
+  buildInputs = [
+    libsepol
+    seadrive-fuse
+    qt5.qtwebengine
+  ] ++ seadrive-fuse.buildInputs;
```

Comment:
Is this semantically correct, i.e. if seadrive-gui needs seadrive-fuse in `PATH`, will it really always need the same buildInputs/libraries?

Reply:
Oh. I see that...

It's still not clear if they are superset because of a coincidence or because that is semantically what it happening in the underlying build system

I'd have to check upstream, but I won't have time today

Reply:
(fwiw I was the one telling to do so)

I'd argue that this reduces redundancy, a change in the deps of `-fuse` is likely to affect `-gui` as well I think.